### ISSUE TITLE
Ability to process images in parallel

### ISSUE THREAD

**Issue by danhstevens** [peterbraden/node-opencv#90]
It seems if I try to process several images at the same time (for example, using the npm async's parallel method) that images are still processed one at a time. Is this intentional? Is there a way to process several images simultaneously?

**Comment by danschultzer** [comment]
I've opencv running in parallel successfully. Since this issue is more than two years old, I'll assume that it's no longer relevant. Feel free to reply if you still have trouble, and I'll reopen it.
